app/models/participant_declaration/ecf.rb

```diff
@@ -16,10 +18,10 @@ def uplift_paid?
       (sparsity_uplift || pupil_premium_uplift)
   end
 
-  def set_temp_type
```

Comment:
shouldn't we continue setting temp type, otherwise how will new declarations get this value set 🤔